modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -187,7 +215,7 @@ You can also add arbitrary tolerations to daemon sets.
 The Taint-Based Evictions feature, enabled by default, evicts pods from a node that experiences specific conditions, such as `not-ready` and `unreachable`.
 When a node experiences one of these conditions, {product-title} automatically adds taints to the node, and starts evicting and rescheduling the pods on different nodes.
 
-Taint Based Evictions has a `NoExecute` effect, where any pod that does not tolerate the taint will be evicted immediately and any pod that does tolerate the taint will never be evicted.
+Taint Based Evictions have a `NoExecute` effect, where any pod that does not tolerate the taint will be evicted immediately and any pod that does tolerate the taint will never be evicted.
```

Comment:
Should _Based Evictions_ be capitalized?

Reply:
Not sure. it is different all over the place. I removed the caps and added a hypen: _taint-based evictions_.